src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -0,0 +1,62 @@
+package mate.academy.spring.controller;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.model.dto.MovieSessionRequestDto;
+import mate.academy.spring.model.dto.MovieSessionResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.mapper.MovieSessionDtoMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movie-sessions")
+public class MovieSessionController {
+    private final MovieSessionService movieSessionService;
+    private final MovieSessionDtoMapper movieSessionDtoMapper;
+
+    @Autowired
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionDtoMapper movieSessionDtoMapper) {
+        this.movieSessionService = movieSessionService;
+        this.movieSessionDtoMapper = movieSessionDtoMapper;
+    }
+
+    @GetMapping("/available?movieId={movieId}&date={date}")
+    public List<MovieSessionResponseDto> getByParameters(@PathVariable Long movieId,
+                                                         @PathVariable LocalDate date) {
```

Comment:
```suggestion
    @GetMapping("/available")
    public List<MovieSessionResponseDto> getByParameters(@RequestParam Long movieId,
                              @RequestParam @DateTimeFormat(pattern = "dd.MM.yyyy") LocalDate date) {
```
It looks like harcoding. I think we need to use here @RequestParam instead @PathVariable.
Also you can use pattern with @DateTimeFormat(pattern = "dd.MM.yyyy", see hints in read.me

Reply:
ok